Library/brew.sh

```diff
@@ -57,7 +57,7 @@ then
   unset HOMEBREW_RUBY_PATH
 fi
 
-HOMEBREW_SYSTEM="$(uname -s)"
+HOMEBREW_SYSTEM="$(/usr/bin/uname -s)"
```

Comment:
I would suggest to use `uname` directly. Consider we haven't seen any breakage caused by user's uname from PATH.